### PR TITLE
Add definition for Faker.js

### PR DESCRIPTION
**Pull request by joeybaker**
Creates a new definition for [faker.js](https://github.com/Marak/faker.js)

### CODE REVIEW

File: definitions/npm/faker_v4.x.x/flow_v0.25.x-/faker_v4.x.x.js [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+      paragraph: (sentenceCount?: number) => string,
+      paragraphs: (paragraphCount?: ?number, separator?: string) => string,
+      text: (times?: number) => string,
+      lines: (lineCount?: number) => string
+    },
+    name: {
+      firstName: (gender?: mixed) => string,
+      lastName: (gender?: mixed) => string,
+      findName: (
+        firstName: string,
```
Comment (JelteF) [inline definitions/npm/faker_v4.x.x/flow_v0.25.x-/faker_v4.x.x.js]: this should be `firstName?: ?string,`, because it's optional

Comment (joeybaker) [reply definitions/npm/faker_v4.x.x/flow_v0.25.x-/faker_v4.x.x.js]: Huh. The docs indicate it's required: https://marak.github.io/faker.js/faker.name.html#-static-findName__anchor, but the code shows that its not: https://github.com/Marak/faker.js/blob/3a4bb358614c1e1f5d73f4df45c13a1a7aa013d7/lib/name.js#L68-L77

Thanks! Will change.

Comment (joeybaker) [reply definitions/npm/faker_v4.x.x/flow_v0.25.x-/faker_v4.x.x.js]: Done.

File: definitions/npm/faker_v4.x.x/test_faker_v4.x.x.js [hunk 2]
```diff
@@ -0,0 +1,311 @@
+import faker from "faker";
```
Comment (AndrewSouthpaw) [inline definitions/npm/faker_v4.x.x/test_faker_v4.x.x.js]: Move the test file into the versioned flow directory.

File: definitions/npm/faker_v4.x.x/test_faker_v4.x.x.js [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+
+faker.address.zipCode();
```
Comment (AndrewSouthpaw) [inline definitions/npm/faker_v4.x.x/test_faker_v4.x.x.js]: Use `it/describe` blocks for more winning.

**Comment by joeybaker** [comment]
@AndrewSouthpaw thanks for the review! I moved the file and added a few `describe` blocks. I'm not sure there's much benefit to breaking down the tests more.

**Pull request merged**